cmake/BuildDynLib.cmake

```diff
@@ -1,10 +1,7 @@
 cmake_minimum_required(VERSION 3.14)
 
-if(MSVC)
-    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
-endif()
+add_library(dynlib SHARED test/myDynLib.cpp)
 
-add_library(dynlib SHARED
-        test/myDynLib.cpp)
+target_include_directories(dynlib PUBLIC .)
```

Comment:
I'm not sure this is useful? I could be wrong though.  
It is at the very least confusing on my perspective as I don't see any header files in the `cmake/` directory

Reply:
It is necessary to compile the dynamic library, it requires DyLib.hpp to compile